tests/Console/Command/FixCommandTest.php

```diff
@@ -29,6 +30,18 @@
  */
 final class FixCommandTest extends TestCase
 {
+    public function testIntersectionPathMode(): void
+    {
+        $cmdTester = $this->doTestExecute([
+            '--path-mode' => 'intersection',
```

Comment:
```suggestion
            '--path-mode' => 'intersection',
            '--show-progress' => 'none',
```

just to make sure which `if` branch we enter